linker/shared/src/main/scala/org/scalajs/linker/frontend/optimizer/IncOptimizer.scala

```diff
@@ -259,6 +264,47 @@ final class IncOptimizer private[optimizer] (config: CommonPhaseConfig, collOps:
 
   }
 
+  /** Elidable constructors: compute the fix point of hasElidableConstructors.
+   *
+   *  ELIDABLE CTORS PASS ONLY. (This IS the elidable ctors pass).
+   */
+  private def updateElidableConstructors(): Unit = {
+    import ElidableConstructorsInfo._
+
+    /* Invariant: when something is in the stack, its
+     * elidableConstructorsInfo was set to NotElidable.
+     */
+    val toProcessStack = mutable.ArrayBuffer.empty[Class]
```

Comment:
Use `mutable.ArrayStack` for readability? (then call `push` / `pop`).

Reply:
`ArrayStack` does not exist in 2.13.x anymore, unfortunately.